_posts/2018-02-16-NewPipe-Beta-now-available.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: NewPip Beta now available
+short: NewPipe Beta published
+date: 2018-02-16 16:00:00 +0100
+categories: [announcement, pinned]
+author: TobiGr
+image: newpipe-beta
+excerpt_separator: <!-- more -->
+---
+
+With the aim of making NewPipe bug free and achieving a stable 1.0.0 release,
+we are pleased to announce the immediate availability of NewPipe Beta.
+
+<!-- more -->
+
+The beta app can be downloaded and installed via [F-Droid](https://f-droid.org/packages/org.schabi.newpipe.beta/) as a separate app.
```

Comment:
I think we have to reword this sentence a bit. It's best to have the link at the end.

How about "The beta is provided as a separate app and can be ..."?

Reply:
Sounds good.